composer.json

```diff
@@ -5,11 +5,11 @@
     "homepage": "https://spryker.com",
     "license": "MIT",
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.0",
```

Comment:
Our tooling is usually on purpose 1-2 minors behind (so it can be used also to work on older versions of Spryker code).
Is there a reason we need to do this here?

Reply:
Yes. To be honest, this is really not necessary for our tools.

Reply:
We should focus on only whats absolutely necessary at this point
You can however at this point make it 7.4+; thats fine as we now go 8.0 (1 minor behind).

Reply:
@dereuromark Ok, no problem. So I will do it for all our packages/tools like this (code-sniffer, development-core)?

Reply:
dev core is core as it is directly shipped as module internally.
I am talking about the normal tooling as per public visibility.